contracts/libraries/RandaoLib.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./RLPReader.sol";
+
+/// @title RandaoLib
+/// @notice Handles Randao related operations
+library RandaoLib {
+    using RLPReader for RLPReader.RLPItem;
+    using RLPReader for RLPReader.Iterator;
+    using RLPReader for bytes;
+
+    /// @notice Get the Randao mixDigest from the header
+    /// @param _item The RLP data of the header
+    /// @return The Randao mixDigest
+    function getRandaoFromHeader(RLPReader.RLPItem memory _item) internal pure returns (bytes32) {
```

Comment:
Is it applied here that internal functions are prefixed with an underscore?

Reply:
According to the libraries in opstack, there is no underscore prefixed on the internal functions